### PR TITLE
fix: use chunk for classnames & fix ReactQuery in `1.2.30`

### PR DESCRIPTION
**Pull request by rxri**

### CODE REVIEW

File: jsHelper/spicetifyWrapper.js [hunk 1]
```diff
@@ -574,8 +593,13 @@ window.Spicetify = {
 	}
 
 	const dropdownChunk = Object.entries(require.m).find(([, value]) => value.toString().includes("dropDown") && value.toString().includes("isSafari"));
-	if (dropdownChunk) {
-		Spicetify.ReactComponent.Dropdown = Object.values(require(dropdownChunk[0])).find(m => typeof m === "function");
+	if (dropdownChunk) Spicetify.ReactComponent.Dropdown = Object.values(require(dropdownChunk[0])).find(m => typeof m === "function");
```
Comment (SunsetTechuila) [inline jsHelper/spicetifyWrapper.js:596]: there is no reason to make this one line
you just make readability worse

File: jsHelper/spicetifyWrapper.js [hunk 2]
```diff
@@ -566,6 +578,13 @@ window.Spicetify = {
 		}
 	});
 
+	// classnames
+	// https://github.com/JedWatson/classnames/
+	const classnamesChunk = Object.entries(require.m).find(
+		([_, value]) => value.toString().includes("[native code]") && !value.toString().includes("<anonymous>")
+	);
+	if (classnamesChunk) Spicetify.classnames = Object.values(require(classnamesChunk[0])).find(m => typeof m === "function");
```
Comment (SunsetTechuila) [inline jsHelper/spicetifyWrapper.js:586]: for consistency 

```suggestion
	if (classnamesChunk) {
		Spicetify.classnames = Object.values(require(classnamesChunk[0])).find(m => typeof m === "function");
	}
```

**Review by SunsetTechuila — CHANGES_REQUESTED** [review]

File: jsHelper/spicetifyWrapper.js (hunk 1 above)
Comment (rxri) [reply jsHelper/spicetifyWrapper.js:596]: you literally said if it's in one line then it's fine, now you change your mind

**Comment by SunsetTechuila** [comment]
there is a clear visual pattern, why are you trying to break it?

image

File: jsHelper/spicetifyWrapper.js (hunk 1 above)
Comment (SunsetTechuila) [reply jsHelper/spicetifyWrapper.js:596]: because this case is special https://github.com/spicetify/spicetify-cli/pull/2792#issuecomment-1920174012

**Comment by rxri** [comment]
ok whatever, idk what's your deal on some "clear visual pattern" but whatever at this point

**Review by kyrie25 — APPROVED** [review]

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**